eslint.config.mjs

```diff
@@ -0,0 +1,54 @@
+import globals from "globals";
+import js from '@eslint/js';
+import tsParser from '@typescript-eslint/parser';
```

Comment:
Out of curiosity, `typescript-eslint` recommends using their own `.config` fn, would this be useful? https://typescript-eslint.io/getting-started